### PR TITLE
[MINOR]: Update create_window_expr to refer only input schema

### PR DESCRIPTION
**Pull request by mustafasrepo**
## Which issue does this PR close?

Closes #.

## Rationale for this change
With the [PR8920](https://github.com/apache/arrow-datafusion/pull/8920) `create_window_expr` function refers to window_schema to get its window expr output type. However, at this stage it should calculate its output type using `input_schema`. With this PR, function only refers to its input schema.

### CODE REVIEW

**Review by alamb — APPROVED** [review]
THank you @mustafasrepo  -- this makes sense to me. cc @comphead

**Review by viirya — APPROVED** [review]

**Comment by alamb** [comment]
cc @comphead

**Comment by alamb** [comment]
I recommend we wait for another day to see if @comphead  has any additional comments and then merge this PR

File: datafusion/core/src/physical_planner.rs [hunk 1]
```diff
@@ -1552,10 +1549,10 @@ pub fn create_window_expr_with_name(
     e: &Expr,
     name: impl Into<String>,
     logical_input_schema: &DFSchema,
+    physical_input_schema: &Schema,
```
Comment (comphead) [inline datafusion/core/src/physical_planner.rs:1552]: Thanks @mustafasrepo 
I'm still thinking if we need to have a physical schema? This is misleading a bit, we always need to keep in mind why the schemas are not the same, what is their difference, etc.

Basically we have a check earlier in the planner that checks actual schemas from batch and from the planner

Comment (mustafasrepo) [reply datafusion/core/src/physical_planner.rs]: Thanks @comphead, It seems that we really don't need physical_input_schema. Removed it, thanks for the suggestion.

**Pull request merged**